### PR TITLE
Allow strings for column names

### LINKED ISSUE #51: Allow strings for column names
`freqtable(df, "var")` gives a method errod. 

Now that DataFrames allows for `String` indexing it would be nice if this did as well.

### PR DESCRIPTION
**Pull request by nalimilan**
Since these are allowed by DataFrames.

Fixes #51.

### CODE REVIEW

**Review by bkamins — APPROVED** [review]

**Comment by bkamins** [comment]
Thank you! The only question is what happens when you do `setdimnames!` - are then they always converted to `Symbol`? (I guess yes, but I do not see such tests)

**Comment by nalimilan** [comment]
Good catch. The input type was preserved, but in case of mixed type better convert everything to a common type.

File: src/freqtable.jl [hunk 1]
```diff
@@ -191,10 +191,11 @@ freqtable(x::AbstractCategoricalVector...; skipmissing::Bool = false,
           subset::Union{Nothing, AbstractVector{Int}, AbstractVector{Bool}} = nothing) =
     _freqtable(x, skipmissing, weights, subset)
 
-function freqtable(t, cols::Symbol...; args...)
+function freqtable(t, cols::Union{Symbol, AbstractString}...; args...)
     all_cols = Tables.columns(t)
-    a = freqtable((Tables.getcolumn(all_cols, y) for y in cols)...; args...)
-    setdimnames!(a, cols)
+    a = freqtable((Tables.getcolumn(all_cols, Symbol(y)) for y in cols)...; args...)
+    cols′ = all(x -> x isa AbstractString, cols) ? cols : Symbol.(cols)
```
Comment (bkamins) [inline src/freqtable.jl:197]: nice

**Comment by nalimilan** [comment]
https://github.com/JuliaRegistries/General/pull/22847

**Pull request merged**